api/app/lib/statistics.rb

```diff
@@ -0,0 +1,104 @@
+require "google/apis/analytics_v3"
+require "google/api_client/auth/key_utils"
+
+class Statistics
+  include ActiveModel::Validations
+  include ActiveModel::Conversion
+  extend ActiveModel::Naming
+
+  def id
+    "statistics-set"
+  end
+
+  def initialize(_range = nil)
```

Comment:
Do you do anything with _range? It'd be better if I could pass a range to this object when I initialize it.